drivers/flash/nrf_qspi_nor.c

```diff
@@ -1126,9 +1199,12 @@ static int qspi_nor_pm_action(const struct device *dev,
 			return -EIO;
```

Comment:
The resume pm implementation is not working. Try this one: The problem is when the device wakes up from sleep QSPI is not working due to clock mismatching. 
`
case (DEVICE_PM_ACTIVE_STATE):
                    **ret = qspi_nrfx_configure(dev);**

                    if (ret != 0) {
                        return -EIO;
                    }

					ret = exit_dpd(dev, false);
					if (ret < 0) {
						return ret;
					}
					ANOMALY_122_UNINIT(dev);
				break;
`
				
For me exit and enter from acquiring and release does not make QSPI sleep. We have to call pm API's from application.

Reply:
The only change this PR makes to the PM_DEVICE actions within the qspi driver here is to disable enter/exit DPD on PM_DEVICE actions if IDLE_IN_DPD is set. This PR wouldn't change anything regarding the QSPI clock.

I'd be happy to get a fix into this PR if it's simple, but our project isn't using PM_DEVICE right now, so I don't have a good way to test this to figure out what's wrong.